modules/udp_bridge.py

```
@@ -0,0 +1,183 @@
+"""
+This module acts as a UDP bridge between rabbit mq networks
+"""
+import pickle
+import time
+import threading
+import sys
+
+from modules.utils import (
+    udp,
+    load_config,
+    cli_parser,
+    async_message,
+    message_broker,
+    excepthook,
+)
+
+from modules.__context import definitions
+from definitions import data_types
+
+
+class Message_Producer(async_message.Async_Threaded_Queue):
+    def __init__(self, host):
+        super().__init__()
+        self._host = host
+
+    def interpret_msg(self, timestanmp, msg):
+        if not isinstance(msg, data_types.Telemetry_Message):
+            return  # TODO: Err here
```

Simple todo, plz fix. Silent errors make me cry.